README.md

```diff
@@ -1,12 +1,18 @@
 # @brave-intl/currency
 
-a currency conversion ratio cache
+a currency conversion ratio cache. 
+Atual version is 1.4.6 
 
 ## installation
 
 ```sh
 npm install --save @brave-intl/currency
 ```
+## Depenencies 
+```ava": ^1.2.1,
```

Comment:
i don't think this section is necessary since they are referenced in the `devDependencies` in `package.json`

what would be helpful is an explanation of how docker could be used instead of setting up psql and how to run migrations either via docker or straight from the sql files